commands/music.py

```diff
@@ -319,53 +325,67 @@ async def _leave(self, ctx: commands.Context):
         await ctx.voice_state.stop()
         del self.voice_states[ctx.guild.id]
 
-    @commands.command(name='volume')
-    async def _volume(self, ctx: commands.Context, *, volume: int):
-        """Sets the volume of the player."""
+    @commands.command(name='volume', aliases=['vol'])
+    async def change_volume(self, ctx, *, vol: float):
+        """Change the player volume.
+        Parameters
 
-        if not ctx.voice_state.is_playing:
-            return await ctx.send('Nothing being played at the moment.')
+        volume: float or int [Required]
+            The volume to set the player to in percentage. This must be between 1 and 100.
+        """
+        vc = ctx.voice_client
 
-        if 0 > volume > 100:
-            return await ctx.send('Volume must be between 0 and 100')
+        if not vc or not vc.is_connected():
+            return await ctx.send('I am not currently connected to voice!', delete_after=20)
 
-        ctx.voice_state.volume = volume / 100
-        await ctx.send('Volume of the player set to {}%'.format(volume))
+        if not 0 < vol < 101:
+            return await ctx.send('Please enter a value between 1 and 100.')
 
-    @commands.command(name='now', aliases=['current', 'playing'])
-    async def _now(self, ctx: commands.Context):
-        """Displays the currently playing song."""
+        player = self.get_player(ctx)
 
-        await ctx.send(embed=ctx.voice_state.current.create_embed())
+        if vc.source:
+            vc.source.volume = vol / 100
+
+        player.volume = vol / 100
+        await ctx.send(f'**`{ctx.author}`**: Set the volume to **{vol}%**')
```

Comment:
To me this feels like it will look weird with "Set" having an uppercase 'S' as it isn't the first word if the sentence. Depends on if this is supposed to operate as an announcement that the user took the action, a statement on behalf of the user, or a reply to the user

Reply:
This is resolved if volume is removed